run_all_experiments.py

```diff
@@ -42,6 +42,7 @@
 
 BENCHMARK_DIR: str = './benchmark-sets/comparison'
 RESULTS_DIR: str = run_one_experiment.RESULTS_DIR
+GENERATED_BENCHMARK_DIR: str = 'generated-benchmark-'
```

Comment:
Could you keep this under the current `benchmkar-sets/` dir, so that all benchmarks are under it? E.g.,
```python
os.path.join('benchmark-sets', 'generated-')
```

Reply:
Yes, done